nix_review/cli/__init__.py

```diff
@@ -62,6 +62,12 @@ def rev_flags(subparsers: argparse._SubParsersAction) -> argparse.ArgumentParser
     rev_parser.add_argument(
         "commit", help="commit/tag/ref/branch in your local git repository"
     )
+    rev_parser.add_argument(
+        "-r",
+        "--remote",
+        default="https://github.com/NixOS/nixpkgs",
+        help="Name of the nixpkgs repo to review",
+    )
```

Comment:
I think it would also make sense to add this to `wip` as well.

Reply:
You're absolutely right. Fixed.